.github/workflows/test.yml

```diff
@@ -500,7 +500,8 @@ jobs:
       - name: Lint with golangci-lint
         uses: reviewdog/action-golangci-lint@v2
         with:
-          filter_mode: "added"
+          fail_level: none
```

Comment:
I suggest we downgrade rules we want to be less strict on to warnings rather than disabling them entirely. As someone who reviews many of these PRs, this change means I’ll now have to manually assess each case to see what’s being ignored, rather than relying on a consistent standard for what we’ve agreed to downgrade.

```
linters-settings:
  nestif:
    severity: warning
```

Reply:
For one off exceptions, maybe we do:

```
      - name: Set fail level based on labels
        id: fail_level
        run: |
          if gh pr view ${{ github.event.pull_request.number }} --json labels --jq '.labels | any(.name == "lint-ignore")'; then
            echo "FAIL_LEVEL=none" >> $GITHUB_ENV
          else
            echo "FAIL_LEVEL=warning" >> $GITHUB_ENV
          fi
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

      - name: Lint with golangci-lint
        uses: reviewdog/action-golangci-lint@v2
        with:
          filter_mode: "diff_context"
          fail_level: ${{ env.FAIL_LEVEL }}
```